webapp/src/main/webapp/WEB-INF/messages.properties

```diff
@@ -506,6 +506,7 @@ Concept.find=Find Concept(s)
 Concept.retiredMessage=This concept is retired
 Concept.name=Concept Name
 Concept.name.atLeastOneRequired=At least one non-empty name is required
+Concept.description.atLeastOneRequired=At least one non-empty Description is required
 Concept.shortName.help=The text shown in report headers and other places where space is limited
```

Comment:
@wluyima Removed the message(Concept.Description.Existing.Required=Existing Description can't be Empty)
as I am not complaining when a user tries to save a concept with empty description,but only checking if there is atleast one description in any locale.
Updated the ConceptSaveHandler to remove any empty descriptions.